apiserver/params/status.go

```diff
@@ -42,15 +42,33 @@ type MachineStatus struct {
 	AgentStatus    DetailedStatus `json:"agent-status"`
 	InstanceStatus DetailedStatus `json:"instance-status"`
 
-	DNSName    string                    `json:"dns-name"`
-	InstanceId instance.Id               `json:"instance-id"`
-	Series     string                    `json:"series"`
-	Id         string                    `json:"id"`
-	Containers map[string]MachineStatus  `json:"containers"`
-	Hardware   string                    `json:"hardware"`
-	Jobs       []multiwatcher.MachineJob `json:"jobs"`
-	HasVote    bool                      `json:"has-vote"`
-	WantsVote  bool                      `json:"wants-vote"`
+	DNSName string `json:"dns-name"`
+
+	// IPAddresses holds the IP addresses bound to this machine.
+	IPAddresses []string `json:"ip-addresses"`
```

Comment:
As discussed in IRC, we are adding another field to the struct, but not changing any existing.